src/MoBi.Presentation/Tasks/Edit/EditTasksForSimulation.cs

```diff
@@ -132,23 +131,28 @@ public void CalculateScaleFactors(IMoBiSimulation simulation)
             if (command.IsEmpty())
                return;
 
-            AddCommand(command);
+            addCommand(command);
          }
       }
 
+      public void Configure(IMoBiSimulation simulation)
+      {
+         throw new System.NotImplementedException();
```

Comment:
Is this PR ready to merge or should I wait for more upcoming commits?

Reply:
This method should be gone. THat's where I started the implementation. Good catch. Will remove and update caption

Reply:
This is so weird. I don't see this code... hum going to update again

Reply:
oh it's because there is EditTaskForSimulation and EditTasksForSimulation in MoBi...